### PR TITLE
fix lint issues [Android]

### PR DESCRIPTION
**Pull request by andresesfm**
Includes deprecation warnings

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -2,11 +2,11 @@
 buildscript {
   repositories {
     google()
-    jcenter()
```
Comment (hyochan) [inline android/build.gradle:5]: Could you verify that this change is correct? Previously, this change made the module broken and fixed in #1328

File: android/build.gradle [hunk 2]
```diff
@@ -41,8 +41,8 @@ android {
 }
 
 repositories {
-  mavenCentral()
   google()
+  mavenCentral()
```
Comment (hyochan) [inline android/build.gradle:45]: What is the actual effect of changing this order?

File: android/build.gradle (hunk 1 above)
Comment (andresesfm) [reply android/build.gradle:5]: Hello @hyochan, I noticed that, thank you for fixing it last time. This time though, I have made sure everything works by re-running the new CI check for android https://github.com/dooboolab/react-native-iap/pull/1333/checks?check_run_id=2602685566 now we can move faster with more confidence

File: android/build.gradle (hunk 2 above)
Comment (andresesfm) [reply android/build.gradle:45]: Just a bit better speed when building

**Review by hyochan — APPROVED** [review]
LGTM!

**Pull request merged**